frontend/contribute.html

```diff
@@ -231,8 +231,8 @@ <h1><!--Section A - -->Dataset</h1>
 
                         <div class="rodi_section_02">
                             <div class="row">
-                                <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2"></div>
-                                <div class="col-xs-12 col-sm-8 col-md-8 col-lg-8">
+
+                                <div class="col-xs-12 col-sm-12 col-md-10 col-md-push-2 col-lg-8">
```

Comment:
Likewise. On `sm`, it grows from 8 columns to columns.

Reply:
Here I found some problems  in this file that i have not yesterday... Now I correct my branch, but my branch now is another.... May I change my branch in webstorm and push html file again to solve the problems ? or I must make another branch and push again?.... thx

Reply:
You can switch branch whenever you want. What you commit will be saved in the current branch you are on.